BackofficeBundle/Controller/BlockController.php

```diff
@@ -41,6 +42,7 @@ public function formAction(Request $request, $nodeId, $blockNumber = 0)
 
         if ($node) {
             $editionRole = $node->getNodeType() === NodeInterface::TYPE_ERROR? TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_ERROR_NODE:TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_NODE;
+            $editionRole = $node->getNodeType() === NodeInterface::TYPE_TRANSVERSE? TransverseNodePanelStrategy::ROLE_ACCESS_UPDATE_GENERAL_NODE:$editionRole;
```

Comment:
i think is more explicit create a protected method getEditionRole with if inside like in Node Controller